platform/pulpcore/app/models/progress.py

```diff
@@ -60,7 +60,7 @@ class ProgressReport(Model):
     total = models.IntegerField(null=True)
     done = models.IntegerField(default=0)
 
-    task = models.ForeignKey("Task", related_name='progress_reports', on_delete=models.CASCADE)
```

Comment:
Looks like a few other unintended changes happened in the rebase.  Reviewed the diff and think I found them all.